share/misc/pci_vendors

```diff
@@ -92,7 +92,7 @@
 0200  Dell (wrong ID)
 # 021b is not Compaq but there is a board misprogrammed
 021b  Compaq Computer Corporation
-	8139  HNE-300 (RealTek RTL8139c) [iPaq Networking]
+	8139  HNE-300 (Realtek RTL8139c) [iPaq Networking]
```

Comment:
This file, sadly, cannot be changed as we get it from elsewhere. Normally, we have all those files under contrib, but this one is a special case, so I dropped it from the commit I landed.